angular.json

```diff
@@ -23,7 +23,15 @@
                         "polyfills": "src/polyfills.ts",
                         "tsConfig": "tsconfig.app.json",
                         "aot": true,
-                        "assets": ["src/favicon.ico", "src/assets"],
+                        "assets": [
+                          "src/favicon.ico",
+                          "src/assets",
+                          {
+                            "glob": "**/*",
+                            "input": "./node_modules/@pxblue/icons-svg",
+                            "output": "./assets/@pxblue/icons-svg"
+                          }
+                        ],
```

Comment:
Do we still need this if we're not going to use the icons?

Reply:
You're right; I've just removed this.